src/runtime/composables/useClientStripe.ts

```diff
@@ -9,5 +9,14 @@ import { useNuxtApp } from "#imports"
  */
 
 export default function useClientStripe() {
+  const { public: { stripe: { publishableKey } } } = useRuntimeConfig()
+
+  // Check status publishableKey
+  if (!publishableKey) {
+    throw new Error('Missing publishableKey option.')
+  } else {
+    console.info('Stripe publishableKey loaded successfully.');
+  }

```

Comment:
@Sandros94 let's remove the `console.info`, it's not bringing so much value, wdty?

Reply:
True